src/EnergyPlus/BaseboardElectric.cc

```diff
@@ -403,26 +393,12 @@ namespace BaseboardElectric {
         // SUBROUTINE INFORMATION:
         //       AUTHOR         Richard Liesen
         //       DATE WRITTEN   Nov 2001
-        //       MODIFIED       na
-        //       RE-ENGINEERED  na
 
         // PURPOSE OF THIS SUBROUTINE:
         // This subroutine initializes the Baseboard units during simulation.
 
         auto &baseboard = state.dataBaseboardElectric;
 
-        // need to check all units to see if they are on ZoneHVAC:EquipmentList or issue warning
-        if (!baseboard->baseboards(BaseboardNum).ZoneEquipmentListChecked && state.dataZoneEquip->ZoneEquipInputsFilled) {
```

Comment:
Moved error message into `GetZoneEquipControlledZoneNum` function which is used during input processing, so this init check is no longer needed.